src/Runtime/Runtime/System.Windows.Browser/HtmlDocument.cs

```diff
@@ -93,6 +93,11 @@ public HtmlElement GetElementById(string id)
             return null;
         }
 
+        public override void SetProperty(string name, object value)
+        {
+            OpenSilver.Interop.ExecuteJavaScript($"document.querySelector('{name}').textContent='{value}'");
```

Comment:
This implementation will work for any property defined on 'document' while the previous one is basically only working for the title property.
```suggestion
            string sValue = CSHTML5.INTERNAL_InteropImplementation.GetVariableStringForJS(value);
            OpenSilver.Interop.ExecuteJavaScriptVoid($"document['{name}'] = {sValue}");
```